config/application.rb

```diff
@@ -31,6 +31,7 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+    config.load_defaults = 7.0
```

Comment:
This PR isn't doing anything - we need to call the `#load_defaults` method!

```suggestion
    config.load_defaults 7.0
```